### ISSUE TITLE
ngPlaceHolder with ngMinDate problem

### ISSUE THREAD

**Issue by rglombovsky** [jfmdev/ngComboDatePicker#35]
Hello! 
I'm posting here because I can't find a solution to the following problem.
I'm trying to use ngPlaceHolder along with ngMinDate, but it's not working.
Instead of showing a placeholder, it shows the min date possible.
If I delete ngMinDate, I can see the placeholder without problem.

With ngMinDate
```
<ng-combo-date-picker name="birthdate" ng-model="birthdate" ng-placeholder="Year,Month,Day"
                                              ng-min-date="{{ $ctrl.validate.minDate($ctrl.passenger) }}">
                        </ng-combo-date-picker>
```

![date](https://user-images.githubusercontent.com/19272522/27140358-b1329b00-50fb-11e7-8907-1477900e01b3.png)

Without ngMinDate:

```
<ng-combo-date-picker name="birthdate" ng-model="birthdate" ng-placeholder="Year,Month,Day">
                        </ng-combo-date-picker>
```

![dateplace](https://user-images.githubusercontent.com/19272522/27140488-08e9f438-50fc-11e7-9cbb-4c89da37830d.png)

Thanks!!

**Comment by jfmdev** [comment]
Hi,

Thanks for reporting this bug. I have just published a new version (**1.5.1**) that fix it.  
You can check your example (using the new version) at: https://embed.plnkr.co/diUwjEHyXxEFXFj0ljSl/

Regards

**Comment by rglombovsky** [comment]
Thanks!
